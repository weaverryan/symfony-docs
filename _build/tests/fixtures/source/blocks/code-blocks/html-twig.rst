
.. code-block:: html+twig
    {# some code #}
    <!-- some code -->
