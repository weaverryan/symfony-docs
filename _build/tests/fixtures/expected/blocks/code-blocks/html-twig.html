<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="literal-block notranslate">
    <div class="highlight-html+twig">
        <table class="highlighttable">
            <tr>
                <td class="linenos">
                    <div class="linenodiv">
                        <pre> 1 2</pre>
                    </div>
                </td>
                <td class="code">
                    <div class="highlight">
                        <pre class="hljs twig">
                            <span class="xml"></span><span class="hljs-comment">{# some code #}</span>
                            <span class="xml">
<span class="hljs-comment">&lt;!-- some code --&gt;</span></span>
                        </pre>
                    </div>
                </td>
            </tr>
        </table>
    </div>
</div>
</body>
</html>
